app/models/service/aggregation.rb

```diff
@@ -42,26 +151,36 @@ def aggregate_direct_vm_memory_on_disk
   end
 
   def aggregate_all_vm_cpus
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
+    attributes["aggregate_all_vm_cpus"] || all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
```

Comment:
Not talking style

For me, using `self["aggregate_all_vm_cpus"]` when the column is not in the query, it blows up for me.
If you use `has_attribute?("aggregate_all_vm_cpus")` and the column is not available, it works.

not intending on talking style